src/InvertMouse/Inverter/InvertMouseBase.cs

```diff
@@ -31,6 +34,36 @@ protected bool IsCursorHidden()
             return (cursorInfo.flags & WinAPI.CURSOR_SHOWING) == 0;
         }
 
+
+        [DllImport("user32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+        static extern IntPtr GetForegroundWindow();
+
+        [DllImport("user32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+        static extern int GetWindowText(IntPtr hWnd, StringBuilder text, int count);
+
+        [DllImport("user32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+        static extern int GetWindowTextLength(IntPtr hWnd);
+
+        bool IsActiveWindowPrefixed()
+        {
+            if (ActiveTitlePrefix == "")
+                return true;
+
+            var handle = GetForegroundWindow();
+            if (handle == null)
+                return false;
+
+            var intLength = GetWindowTextLength(handle) + 1;
```

Comment:
We don't need more `ActiveTitlePrefix.Length` characters from the window title, so we can limit the buffer to that number.